BaseBundle/Tests/AbstractTest/AbstractKernelTestCase.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace OpenOrchestra\BaseBundle\Tests\AbstractTest;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+/**
+ * Class AbstractKernelTestCase
+ */
+class AbstractKernelTestCase extends KernelTestCase
+{
+    use TestCleanUpTearDownTrait;
```

Comment:
You should take care of the kernelTestCase, the tearDown method ensure that the kernel is shutdown